nifi-framework-api/src/test/java/org/apache/nifi/nar/NarManifestTest.java

```diff
@@ -0,0 +1,104 @@
+/*
+ *  Licensed to the Apache Software Foundation (ASF) under one or more
+ *  contributor license agreements.  See the NOTICE file distributed with
+ *  this work for additional information regarding copyright ownership.
+ *  The ASF licenses this file to You under the Apache License, Version 2.0
+ *  (the "License"); you may not use this file except in compliance with
+ *  the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+package org.apache.nifi.nar;
+
+import org.apache.nifi.bundle.BundleCoordinate;
+import org.junit.jupiter.api.Test;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.net.URL;
+import java.util.jar.Manifest;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertNull;
+
+public class NarManifestTest {
+
+    @Test
+    public void testFromManifestFull() throws IOException {
+        final String manifestPath = "nar/MANIFEST-FULL.MF";
+        final Manifest manifest = loadManifest(manifestPath);
+        final NarManifest narManifest = NarManifest.fromManifest(manifest);
+        assertNotNull(narManifest);
+
+        assertEquals("nifi-framework-nar", narManifest.getId());
+        assertEquals("org.apache.nifi", narManifest.getGroup());
+        assertEquals("2.0.0-SNAPSHOT", narManifest.getVersion());
+
```

Comment:
Recommend defining static Strings for these identifier and version values so they can be reused in building objects and in assertions.

Reply:
I can move them to constants, but in this case the values are coming from the actual MANIFEST files in `src/test/resources`, so it is not helping for consistency between building vs assertions.

Reply:
Yea, the main value is just defining it once in the class.